### PR TITLE
Pipe following demo

### PR DESCRIPTION
**Pull request by ignaciotb**

### CODE REVIEW

File: smarc_sensor_plugins/smarc_sensor_plugins_ros/urdf/sonar_snippets.xacro [hunk 1]
```diff
@@ -28,7 +28,7 @@
     </collision>
   </xacro:macro>
 
-  <xacro:macro name="multibeam_intensity_sonar"
+  <xacro:macro name="side_scan_sonar"
```
Comment (nilsbore) [inline smarc_sensor_plugins/smarc_sensor_plugins_ros/urdf/sonar_snippets.xacro:31]: Why change this name? It means we need to change it in all AUV definitions, right?

File: smarc_sensor_plugins/smarc_sensor_plugins_ros/urdf/sonar_snippets.xacro [hunk 2]
```diff
@@ -156,4 +156,75 @@
 	
 	<link name="${namespace}/forward_sonar${suffix}_optical_frame"/>
   </xacro:macro>
+
+  <xacro:macro name="multibeam_intensity_sonar"
```
Comment (nilsbore) [inline smarc_sensor_plugins/smarc_sensor_plugins_ros/urdf/sonar_snippets.xacro:160]: Ok, I don't really know what's going on here. Can you explain, plz?

File: smarc_worlds/launch/random_rocks.launch [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <launch>
-    <arg name="gui" default="true"/>
+    <arg name="gui" default="false"/>
```
Comment (nilsbore) [inline smarc_worlds/launch/random_rocks.launch:2]: These should default to "true".

**Comment by nilsbore** [comment]
There's a lot of moving parts here, please explain what's in the PR.

File: smarc_sensor_plugins/smarc_sensor_plugins_ros/urdf/sonar_snippets.xacro (hunk 1 above)
Comment (ignaciotb) [reply smarc_sensor_plugins/smarc_sensor_plugins_ros/urdf/sonar_snippets.xacro:31]: This we talked about some time ago. I created a new xacro macro for the "multibeam" based on the side_scan one. But the two side scans are still in the urdf

File: smarc_worlds/launch/random_rocks.launch (hunk 3 above)
Comment (ignaciotb) [reply smarc_worlds/launch/random_rocks.launch:2]: right, I'll change it

**Comment by nilsbore** [comment]
This is outdated, closing

**Pull request closed without merging**